examples/asr/asr_cache_aware_streaming/speech_to_text_cache_aware_streaming_infer.py

```diff
@@ -150,16 +152,16 @@ def perform_streaming(asr_model, streaming_buffer, compare_vs_offline=False, deb
                         transcribed_texts,
                         cache_last_channel,
                         cache_last_time,
+                        cache_last_channel_len,
                         previous_hypotheses,
                     ) = asr_model.conformer_stream_step(
                         processed_signal=chunk_audio,
                         processed_signal_length=chunk_lengths,
                         cache_last_channel=cache_last_channel,
                         cache_last_time=cache_last_time,
-                        keep_all_outputs=streaming_buffer.is_buffer_empty(),
+                        cache_last_channel_len=cache_last_channel_len,
```

Comment:
I think keep_all_outputs is necessary for the last batch in regular attention.

Reply:
added back keep_all_outputs, and drop_extra_preencoded